.github/workflows/build-images-action.yml

```diff
@@ -10,6 +10,11 @@ on:
     - 'release-*'
     tags:
     - 'v*'
+  workflow_run:
```

Comment:
This triggers the build images action workflow, yes, but it does not pass context why it was triggered, so it does not know that it needs to build tags.

Reply:
Thanks. I've now added https://github.com/metal3-io/project-infra/pull/927 to account for that.
But now I'm wondering if the "Create Release" wf should be made reusable :-? Wdyt about that?

Reply:
You mean moving it to project-infra? Yes it should, but only after we've ironed out the kinks.